### ISSUE TITLE
Allow CKEditor5 to use arizona-bootstrap CSS

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#2446]
## Problem/Motivation
While enabling CKEditor5 on a local copy of Quickstart 2.7-dev, Drupal 9.5.x I saw a warning message that stated.
![image](https://github.com/az-digital/az_quickstart/assets/1023167/4fa39331-ec15-47e1-aaaf-6cccf48ffefb)

https://www.drupal.org/node/3259165

### Describe the bug
We are currently using the old ckeditor only.

### To Reproduce
Use this composer.json
```
{
    "name": "az-digital/az_quickstart",
    "description": "Arizona Quickstart",
    "type": "drupal-custom-profile",
    "license": "GPL-2.0-only",
    "authors": [
        {
            "name": "Arizona Digital",
            "homepage": "https://digital.arizona.edu",
            "role": "Team"
        }
    ],
    "support": {
        "docs": "https://quickstart.arizona.edu/help"
    },
    "repositories": [
        {
            "type": "composer",
            "url": "https://packages.drupal.org/8"
        },
        {
            "type": "composer",
            "url": "https://asset-packagist.org"
        },
        {
            "type": "vcs",
            "url": "https://github.com/az-digital/phpcs-security-audit",
            "no-api": true
        }
    ],
    "require": {
        "php": ">=7.4",
        "az-digital/arizona-bootstrap": "2.0.23",
        "bower-asset/bootstrap-datepicker": "1.9.0",
        "citation-style-language/locales": "0.0.5",
        "citation-style-language/styles": "0.0.249",
        "composer/installers": "^1.9",
        "cweagans/composer-patches": "^1.7",
        "drupal/access_unpublished": "1.5.0",
        "drupal/antibot": "2.0.2",
        "drupal/asset_injector": "2.17.0",
        "drupal/auto_entitylabel": "3.0.0",
        "drupal/background_image_formatter": "1.7",
        "drupal/better_exposed_filters": "5.0",
        "drupal/block_class": "2.0.11",
        "drupal/block_content_permissions": "1.11.0",
        "drupal/block_field": "1.0.0-rc4",
        "drupal/bootstrap_barrio": "4.30",
        "drupal/bootstrap_utilities": "2.0.0",
        "drupal/cas": "2.1.0",
        "drupal/chosen": "4.0.0",
        "drupal/ckeditor_bs_grid": "2.0.9",
        "drupal/coffee": "1.3.0",
        "drupal/config_distro": "1.0-alpha4",
        "drupal/config_provider": "2.0-rc4",
        "drupal/config_snapshot": "1.0-rc2",
        "drupal/config_sync": "2.0-beta7",
        "drupal/config_update": "1.7",
        "drupal/core-recommended": "9.5.9",
        "drupal/crop": "2.3.0",
        "drupal/ctools": "3.11",
        "drupal/date_ap_style": "1.5.0",
        "drupal/draggableviews": "2.1.3",
        "drupal/easy_breadcrumb": "2.0.5",
        "drupal/embed": "1.6.0",
        "drupal/entity_browser": "2.9.0",
        "drupal/entity_embed": "1.x-dev",
        "drupal/environment_indicator": "4.0.10",
        "drupal/exclude_node_title": "1.4.0",
        "drupal/externalauth": "2.0.3",
        "drupal/field_group": "3.4.0",
        "drupal/field_group_link": "3.1.0",
        "drupal/flag": "4.0-beta4",
        "drupal/google_tag": "1.6.0",
        "drupal/honeypot": "2.1.2",
        "drupal/image_widget_crop": "2.4.0",
        "drupal/inline_entity_form": "1.0.0-rc15",
        "drupal/intelligencebank": "3.0.0",
        "drupal/jquery_ui": "1.6.0",
        "drupal/jquery_ui_datepicker": "1.4.0",
        "drupal/link_class": "2.1.0",
        "drupal/linkit": "6.0.0-rc1",
        "drupal/masquerade": "2.0.0-rc1",
        "drupal/media_entity_file_replace": "1.1",
        "drupal/media_library_edit": "3.0.2",
        "drupal/media_library_form_element": "2.0.6",
        "drupal/media_library_theme_reset": "1.5.0",
        "drupal/media_migration": "1.0.0-alpha16",
        "drupal/menu_block": "1.7.0",
        "drupal/menu_link_attributes": "1.3.0",
        "drupal/metatag": "1.22.0",
        "drupal/migrate_plus": "5.3.0",
        "drupal/migrate_queue_importer": "2.0",
        "drupal/migrate_tools": "5.2.0",
        "drupal/optional_end_date": "1.3.0",
        "drupal/paragraphs": "1.15.0",
        "drupal/paragraphs_admin": "1.4.0",
        "drupal/pathauto": "1.11.0",
        "drupal/quick_node_clone": "1.16.0",
        "drupal/redirect": "1.8.0",
        "drupal/role_delegation": "1.2.0",
        "drupal/search_exclude": "3.0.0-beta1",
        "drupal/seckit": "2.0.1",
        "drupal/slick": "2.7.0",
        "drupal/slick_views": "2.6.0",
        "drupal/smart_date": "3.6.1 || 3.7.2",
        "drupal/smart_title": "1.0-beta1",
        "drupal/smtp": "1.2.0",
        "drupal/token": "1.11.0",
        "drupal/views_bootstrap": "5.4.0",
        "drupal/viewsreference": "2.0.0-beta4",
        "drupal/webform": "6.1.4",
        "drupal/workbench_access": "2.0.1",
        "drupal/xmlsitemap": "1.4.0",
        "jjj/chosen": "2.2.1",
        "npm-asset/blazy": "1.8.2",
        "npm-asset/jquery-ui-touch-punch": "0.2.3",
        "npm-asset/slick-carousel": "1.8.0",
        "seboettg/citeproc-php": "2.6.0"
    },
    "require-dev": {
        "az-digital/az-quickstart-dev": "~1"
    },
    "extra": {
        "branch-alias": {
            "dev-main": "2.7.x-dev"
        },
        "patches": {
            "drupal/antibot": {
                "drupalSettings not updated on rendering form in Ajax (3205217)": "https://www.drupal.org/files/issues/2021-03-24/3205217-1.patch"
            },
            "drupal/auto_entitylabel": {
                "Support entities without bundles (2945387)": "https://www.drupal.org/files/issues/2022-05-20/2945387-19-support-bundleless-entities.patch"
            },
            "drupal/better_exposed_filters": {
                "Remove jquery_ui_slider (3210947)": "https://www.drupal.org/files/issues/2022-02-16/3210947-12.patch"
            },
            "drupal/bootstrap_barrio": {
                "Fix barrio webform required field issue (3133643)": "https://gist.githubusercontent.com/tadean/09ef9559553c788c5637077462b392dd/raw/1190b7d78843af29e2eae31c4601a51e65e5b352/barrio_webform_required_field_issue.patch",
                "Fix schema (3217958)": "https://gist.githubusercontent.com/trackleft/6b7752228979c6932e8d09e01381dd28/raw/70bd7ed90ab487b454a1106aa9935a335b7d4f49/gistfile1.txt",
                "Node template contextual menu issue (3158009)": "https://www.drupal.org/files/issues/2020-07-09/node_contextual.patch"
            },
            "drupal/cas": {
                "ServiceNotFoundException when logging out (2948185)": "https://www.drupal.org/files/issues/cas_logout_error-2948185-2.patch",
                "Drupal 9.2's change to session handling is not compatible with this module (3190842)": "https://www.drupal.org/files/issues/2022-08-04/3190842-only-get-session-id-when-necessary.patch"
            },
            "drupal/config_sync": {
                "Config_sync_test core constraint": "https://gist.githubusercontent.com/tadean/602c6412fbbdcc16a2d8a0fb58642f9e/raw/2ef503a4c0ae7d459f7c789d6ad3720d92945880/config_sync_test_core_requirement_issue.patch"
            },
            "drupal/core": {
                "Rendered content view with relationship issue (2457999)": "https://www.drupal.org/files/issues/2023-01-04/2457999-9.5.x-309.patch",
                "Views Date Filter Datetime Granularity Option (2868014)": "https://www.drupal.org/files/issues/2022-11-16/2868014-127.datetime-granularity.patch",
                "Unnecessary restrictions on logo format: Can't upload replacement SVG logo (2259567)": "https://www.drupal.org/files/issues/2022-05-05/2259567-160-9-3-x.patch",
                "Ajax CSS load order issue (1461322)": "https://www.drupal.org/files/issues/2020-06-05/1461322-25.patch",
                "Layout builder revisions issue (3033516)": "https://www.drupal.org/files/issues/2019-06-17/3033516-17.patch",
                "Allow text field to enforce a specific text format (784672)": "https://www.drupal.org/files/issues/2023-01-05/784672-302.patch",
                "Unable to select single view display for view settings (2552541)": "https://www.drupal.org/files/issues/2020-11-03/2552541-43.patch",
                "MediaEmbed conflates default view mode with user-selected view mode (3109289)": "https://www.drupal.org/files/issues/2022-12-07/drupal_core-mediaembed_default_view_mode-3109289-65.patch",
                "Unrelated error message when running tests with database errors (3163925)": "https://www.drupal.org/files/issues/2022-05-12/3163925-19-9-2-x.patch",
                "Add skip-method option to file_copy and download process plugins (2766369)": "https://www.drupal.org/files/issues/2022-04-23/2766369-33.patch",
                "Move content overview default task link out of content_moderation module (3199682)": "https://www.drupal.org/files/issues/2022-02-10/3199682-11-9-4-x.patch",
                "Changing an existing embedded media does not get saved with CKEditor (3102249)": "https://www.drupal.org/files/issues/2022-09-19/3102249-48.patch",
                "Improve migration system performance: statically cache DrupalSqlBase::$systemData (3154156)": "https://www.drupal.org/files/issues/2022-01-24/3154156-12.patch"
            },
            "drupal/config_distro": {
                "fnmatch issue": "https://www.drupal.org/files/issues/2020-07-04/3144145-replace-fnmatch-with-preg-match-6.patch",
                "missing parent class": "https://git.drupalcode.org/issue/config_distro-3199197/-/commit/c312096d486caa2f01703ca7de1abf2285b6fac8.diff",
                "Storage comparer issue": "https://gist.githubusercontent.com/tadean/4ad67c67a56d9fcec83db8f403ac58d7/raw/9a6762de5b014dffecf06c9a5b311f75c534fce4/config_distro_comparer.patch"
            },
            "drupal/config_provider": {
                "ConfigProviderInterface::addConfigToCreate() @return annotation is wrong (3281977)": "https://www.drupal.org/files/issues/2022-05-23/3281977-2.patch"
            },
            "drupal/date_ap_style": {
                "All Day Setting hides time display (3272760)": "https://www.drupal.org/files/issues/2022-03-31/3272760-7.patch"
            },
            "drupal/entity_embed": {
                "Add support for entity embed in CKEditor5 (3309747)": "https://www.drupal.org/files/issues/2023-05-05/entity_embed-3272732-80.patch"
            },
            "drupal/embed": {
                "CKEditor 5 compatibility (3272732)": "https://www.drupal.org/files/issues/2023-03-30/fix_module_embed_compatibility_CKEditor_5.patch"
            },
            "drupal/media_migration": {
                "Catch migrate RequirementsExceptions (3268480)": "https://www.drupal.org/files/issues/2022-03-08/3268480-2.patch"
            },
            "drupal/migrate_plus": {
                "XML namespacing issue (2933531)": "https://www.drupal.org/files/issues/2019-08-22/register_namespaces-2933531-7.patch"
            },
            "drupal/role_delegation": {
                "Restrict access to cancel/remove accounts with non-assignable roles (3299201)": "https://www.drupal.org/files/issues/2022-07-25/3299201-4.patch"
            },
            "drupal/viewsreference": {
                "Add Label field formatter (2938433)": "https://www.drupal.org/files/issues/2021-01-27/add_label_formatter-2938433-7.patch"
            }
        },
        "drupal-scaffold": {
            "file-mapping": {
                "[web-root]/android-chrome-192x192.png": {
                    "path": "assets/android-chrome-192x192.png",
                    "overwrite": false
                },
                "[web-root]/android-chrome-512x512.png": {
                    "path": "assets/android-chrome-512x512.png",
                    "overwrite": false
                },
                "[web-root]/apple-touch-icon.png": {
                    "path": "assets/apple-touch-icon.png",
                    "overwrite": false
                },
                "[web-root]/browserconfig.xml": {
                    "path": "assets/browserconfig.xml",
                    "overwrite": false
                },
                "[web-root]/favicon-16x16.png": {
                    "path": "assets/favicon-16x16.png",
                    "overwrite": false
                },
                "[web-root]/favicon-32x32.png": {
                    "path": "assets/favicon-32x32.png",
                    "overwrite": false
                },
                "[web-root]/favicon.ico": {
                    "path": "assets/favicon.ico",
                    "overwrite": false
                },
                "[web-root]/mstile-70x70.png": {
                    "path": "assets/mstile-70x70.png",
                    "overwrite": false
                },
                "[web-root]/mstile-144x144.png": {
                    "path": "assets/mstile-144x144.png",
                    "overwrite": false
                },
                "[web-root]/mstile-150x150.png": {
                    "path": "assets/mstile-150x150.png",
                    "overwrite": false
                },
                "[web-root]/mstile-310x150.png": {
                    "path": "assets/mstile-310x150.png",
                    "overwrite": false
                },
                "[web-root]/mstile-310x310.png": {
                    "path": "assets/mstile-310x310.png",
                    "overwrite": false
                },
                "[web-root]/safari-pinned-tab.svg": {
                    "path": "assets/safari-pinned-tab.svg",
                    "overwrite": false
                },
                "[web-root]/site.webmanifest": {
                    "path": "assets/site.webmanifest",
                    "overwrite": false
                }
            }
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "config": {
        "sort-packages": true
    }
}
```
Then enable the ckeditor5 module
Then change `az_standard` text formatter to use CKEditor5 instead of CKEditor

**Comment by trackleft** [comment]
I think we can use the library API, since CKEditor5 does not use an iFrame anymore.
The issue with using the library API in this case has to do with our admin theme depending on our front end theme.

We should consider putting our arizona-bootstrap / az-icons/ proxima nova etc. settings into either a supporting module, or az_core.

**Comment by joeparsons** [comment]
Closing this since we already merged #2447.  Remaining issues will be addressed in #2596.
